sdk/databricks/arm-databricks/README.md

```diff
@@ -1,93 +1,101 @@
-## Azure DatabricksClient SDK for JavaScript
+## Azure AzureDatabricksManagementClient SDK for JavaScript
```

Comment:
that looks bad, how did you regenerate this?

Reply:
Great review. I think the latest codegen leads to some breaking change in generate codes. You also can find the generated codes in https://github.com/AzureSDKAutomation/azure-sdk-for-js/tree/sdkAuto/%40azure_arm-databricks/sdk/databricks/arm-databricks
I will send an email to discuss about this issue.

Reply:
I just check the generated codes with codegen 4.2.2 and the client name has already been changed to `AzureDatabricksManagementClient`. So this breaking change is because the existing codes is too old and it has not been released for long time.

Reply:
@dw511214992 thanks for debugging this. Can you please share the command you used for this generation?

Reply:
the command we used is:` gulp codegen --use @microsoft.azure/autorest.typescript@4.2.2 --package "@azure/arm- databricks"`
you can change the codegen version manually.